libsplinter/src/lib.rs

```diff
@@ -75,6 +75,8 @@ pub mod mesh;
 pub mod network;
 pub mod node_registry;
 pub mod orchestrator;
+#[cfg(any(feature = "rest-api", feature = "scabbard-client"))]
+mod protocol;
```

Comment:
Would it make more sense for the protocol versions to be defined locally for the various REST APIs and clients? It seems odd for these to be centralized when the REST APIs and the clients themselves aren't.

Reply:
The location of these items has yet to be determined, but it is definitely something to be considered (though this approach was based on initial conversations about this feature).